src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Resources/Base/BuildReleaseArm64SimpleDotNet.apkdesc

```diff
@@ -98,5 +98,5 @@
       "Size": 1904
     }
   },
-  "PackageSize": 2689557
+  "PackageSize": 2742805
```

Comment:
Increase by 53KB is somewhat odd, especially as I'm not sure what it's *from*.  Most changes appear negligible: `libmonosgen-2.0.so` is only ~5KB, `System.Private.CoreLib.dll` is only 4KB.

The change appears to be mostly due to `libmonodroid.so`, which is up by 122KB!

*That* may warrant further investigation.  *Why* is `libmonodroid.so` so much larger now?

Reply:
I think it's the effect of two changes - the change to use static methods (which you note farther down) may cause the compiler to more eagerly inline code, and the removal of per-function sections may have caused LTO to inline more code.